src/Generation/race.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from Generation.generator import llm_create
+
+class Hostility(Enum):
```

Comment:
Is this for the labyrinth? I'd say we don't need this for city encounters, since we already have allegience of the faction.

Reply:
Well, I'd say it gives a certain creativity to the generation so the LLM doesn't take only one side of the spectrum of the general, fantasy races.